pkg/receiver/rawk8seventsreceiver/receiver.go

```diff
@@ -103,22 +103,27 @@ func newRawK8sEventsReceiver(
 	restClient := client.CoreV1().RESTClient()
 	for _, namespace := range namespaces {
 		namespaceListWatch := listerWatcherFactory(restClient, "events", namespace, fields.Everything())
-		_, namespaceController = cache.NewInformer(namespaceListWatch, &corev1.Event{}, 0, cache.ResourceEventHandlerFuncs{ // nolint:staticcheck
-			AddFunc: func(obj interface{}) {
-				event := obj.(*corev1.Event)
-				eventCh <- &eventChange{
-					changeType: eventChangeTypeAdded,
-					event:      event,
-				}
+		opts := cache.InformerOptions{
```

Comment:
Should we raise a different PR for this change? I think we'll need a changelog entry for this.

Reply:
In my opinion this is a simpler change than it may seem outwardly. If you have a look at the definition of cache.NewInformer, it just calls cache.NewInformerWithOptions in the same way it's called here. There would be no change to communicate in the changelog.

Reply:
(cache.NewInformer is deprecated, and a notice has been added to switch to cache.NewInformerWithOptions)